src/tests/option_round/option_buyers/refunding_bids_tests.cairo

```diff
@@ -62,7 +65,7 @@ fn place_incremental_bids_internal(
     );
 
     // @dev Bid amounts are each bid price * the number of options available
```

Comment:
need to update this comment or drop it, since each bid amount is the number of options available